gravitee-am-reporter/gravitee-am-reporter-kafka/src/main/java/io/gravitee/am/reporter/kafka/dto/AuditMessageValueDto.java

```diff
@@ -15,198 +15,63 @@
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.Setter;
+
 import java.time.Instant;
 
 /**
  * Copied from File reporter and renamed to match Kafka Semantics
  * @author Florent Amaridon
  * @author Visiativ
  */
+@Setter
+@Getter
 public class AuditMessageValueDto {
 
-  /**
-   * Identifier for a specific audited event
-   */
-  private String id;
-
-  /**
-   * The log transaction id for event correlation
-   */
-  private String transactionId;
-
-  /**
-   * Indicator for type of action performed during the event that generated the audit
-   */
-  private String type;
-
-  /**
-   * The type of the resource who triggered the action
-   */
-  private String referenceType;
-
-  /**
-   * The id of the resource who triggered the action
-   */
-  private String referenceId;
-
-  /**
-   * The access point that performs the event (OAuth client or HTTP client (e.g web browser) with ip address, user agent, geographical information)
-   */
-  private AuditAccessPointDto accessPoint;
-
-  /**
-   * Describes the user, app, client, or other entity who performed an action on a target
-   */
-  private AuditEntityDto actor;
-
-  /**
-   * The entity upon which an actor performs an action
-   */
-  private AuditEntityDto target;
-
-  /**
-   * Indicates whether the event succeeded or failed
-   */
-  private String status;
-
-  /**
-   * The date when the event was logged
-   */
-  private Instant timestamp;
-
-  /**
-   * The environment identifier
-   */
-  private String environmentId;
-
-  /**
-   * The organization identifier
-   */
-  private String organizationId;
-
-  private String domainId;
-
-  private String nodeId;
-
-  private String nodeHostname;
-
-  public String getId() {
-    return id;
-  }
-
-  public void setId(String id) {
-    this.id = id;
-  }
-
-  public String getReferenceType() {
-    return referenceType;
-  }
-
-  public void setReferenceType(String referenceType) {
-    this.referenceType = referenceType;
-  }
-
-  public String getReferenceId() {
-    return referenceId;
-  }
-
-  public void setReferenceId(String referenceId) {
-    this.referenceId = referenceId;
-  }
-
-  public String getType() {
-    return type;
-  }
-
-  public void setType(String type) {
-    this.type = type;
-  }
-
-  public Instant getTimestamp() {
-    return timestamp;
-  }
-
-  public void setTimestamp(Instant timestamp) {
-    this.timestamp = timestamp;
-  }
-
-  public String getTransactionId() {
-    return transactionId;
-  }
-
-  public void setTransactionId(String transactionId) {
-    this.transactionId = transactionId;
-  }
-
-  public AuditAccessPointDto getAccessPoint() {
-    return accessPoint;
-  }
-
-  public void setAccessPoint(AuditAccessPointDto accessPoint) {
-    this.accessPoint = accessPoint;
-  }
-
-  public AuditEntityDto getActor() {
-    return actor;
-  }
-
-  public void setActor(AuditEntityDto actor) {
-    this.actor = actor;
-  }
+    @JsonProperty
```

Comment:
not sure it's needed (see [AuditAccessPointDto](https://github.com/gravitee-io/gravitee-access-management/pull/4268/files#diff-18abfb814783921c68a3df61da906d61e0ec78f51569d2eda90cd70b7b12d2dc))